Libraries/LibWeb/WebAudio/DynamicsCompressorNode.cpp

```diff
@@ -81,11 +81,11 @@ WebIDL::ExceptionOr<void> DynamicsCompressorNode::set_channel_count_mode(Binding
 WebIDL::ExceptionOr<void> DynamicsCompressorNode::set_channel_count(WebIDL::UnsignedLong channel_count)
 {
     if (channel_count > 2) {
-        // Return a NotSupportedError if 'max' is used
+        // Return a NotSupportedError channel count is greater than 2
```

Comment:
Or perhaps I should just delete these comments instead as the code is self-explanatory?

Reply:
Yes